src/com/amazon/ion/impl/bin/IonRawBinaryWriter.java

```diff
@@ -125,14 +123,9 @@
 
     private static final byte VARINT_NEG_ZERO   = (byte) 0xC0;
 
-    // See IonRawBinaryWriter#writeString(String) for usage information.
-    static final int SMALL_STRING_SIZE = 4 * 1024;
```

Comment:
All of the code removed from this file (`IonRawBinaryWriter.java`) was moved to the new `Utf8StringEncoder` class.